Restrictions.cpp

```diff
@@ -0,0 +1,17 @@
+#include <cstring>
+#include "Restrictions.h"
+
+Restrictions::Restrictions(const char *restrictions): restrictions(split(restrictions, " ,")) {}
+
+
+std::unordered_set<std::string> Restrictions::split(const char *restrictions, const char* delimiter) {
+    std::unordered_set<std::string> restrictionsSet;
+    char restrictionsCopy[strlen(restrictions)];
+    strcpy(restrictionsCopy, restrictions);
+    char* restriction = strtok(restrictionsCopy, delimiter);
```

Comment:
Aren't you assuming a space before the ","? Should probably use regex

Reply:
No, I intentionally added the space because it splits by "," and " "